### PR TITLE
modified models/scanvi.py for symmetrical KL option

### PR DESCRIPTION
**Pull request by chenlingantelope**

### CODE REVIEW

File: scvi/models/scanvi.py [hunk 1]
```diff
@@ -204,18 +204,28 @@ def forward(self, x, local_l_mean, local_l_var, batch_index=None, y=None):
                 0.0,
             )
 
-        probs = self.classifier(z1)
+        probs = self.classify(x)
```
Comment (galenxing) [inline scvi/models/scanvi.py:207]: Why the change to self.classify?

Comment (adamgayoso) [reply scvi/models/scanvi.py:207]: It looks like it's for the label_groups thing, though it seems like the implementation is inefficient because it has to encode `x` again.

Comment (chenlingantelope) [reply scvi/models/scanvi.py:207]: Yes this seems like a typo to me, because there is the classify function that deals with both when there is hierarchy information (weight the classifier results by the ancestor probabilities) or not (just returns the results of the classifier)

**Comment by romain-lopez** [comment]
I am unsure whether this is needed anymore. Closed it for now.
